db/migrate/20171104172008_handle_emojis_in_comments.rb

```diff
@@ -0,0 +1,6 @@
+class HandleEmojisInComments < ActiveRecord::Migration
+  def change
+      execute("ALTER TABLE hubstats_comments MODIFY body TEXT CHARACTER SET utf8mb4 COLLATE utf8mb4_bin;")
```

Comment:
@elliott-beach Is all this does modify the type of data stored in that column?

Reply:
Y

Reply:
Also, `set names utf8` sets the connection to use a 4-byte encoding, which I found to be necessary.